specificatie/openapi.yaml

```diff
@@ -557,6 +612,78 @@ components:
         pattern: '^[0-9]{16}$'
         example: '0226010000038820'
 
+    postcode:
+      description: Postcode van het adres.
+      name: postcode
+      in: query
+      required: false
+      schema:
+        type: string
+        pattern: '^[1-9]{1}[0-9]{3}[a-zA-Z]{2}$'
+        example: '6922KZ'
+
+    huisnummer:
+      description: Huisnummer van het adres.
+      name: huisnummer
+      in: query
+      required: false
+      schema:
+        type: integer
+        minimum: 1
+        maximum: 99999
+        example: 1
+
+    huisnummertoevoeging:
+      description: |
+        Een toevoeging (in de vorm van maximaal 4 cijfers of letters) aan een huisnummer of een combinatie van huisnummer en huisletter van het adres.
+        Bijvoorbeeld: 
+        Bij het adres: Belgiëlaan 1 A2, 2391 PH  HAZERSWOUDE-DORP, is 1 het huisnummer, A de huisletter en 2 de huisnummertoevoeging.
+      name: huisnummertoevoeging
+      in: query
+      required: false
+      schema:
+        type: string
+        pattern: '^[0-9a-zA-Z]{1,4}$'
+        example: 'bis'
+
+    huisletter:
```

Comment:
Volgens mij hebben we afgesproken dat query-parameters de post-fix "-query" in de naam krijgen om onderscheid te kunnen maken met path-parameters op dezefde property. Dus huisletter-query.

Reply:
Zie opmerkingen hierboven.